### PR TITLE
accept both ucsc export and true refflat format

### LINKED ISSUE #2453: picard collectRNA-seq metrics should use actual refflat, not ucsc export
It seems that the picard RNA-seq metrics wrapper accepts both GFF/GTF and [refflat](https://genome.ucsc.edu/goldenpath/gbdDescriptionsOld.html#RefFlat)

There are instructions for extracting  UCSC gene info at the bottom of the tool, but this does not generate actual refflat, 

refflat format:
```
DDX11L1 ENST00000456328.2       chr1    +       11868   14409   14409   14409   3       11868,12612,13220,      12227,12721,14409,
```
ucsc extract format:
```
ENST00000456328.2       chr1    +       11868   14409   14409   14409   3       11868,12612,13220,      12227,12721,14409, DDX11L1
```

2 options:
- add a ucsc extract format (and simplify instructions to allow extracting "all fields from the selected table")
- sniff which refflat we have using data in column 4 (numeric = ucsc extract, +/- = refflat)

Which should I implement?
Or a third option I've not considered?
see also #1254

### PR DESCRIPTION
**Pull request by bwlang**
fixes #2453

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by abretaud** [comment]
Hi, could you bump the version number? If it's not too complicated, a test case would be great also

**Comment by bwlang** [comment]
I think this is ready to merge

File: tools/picard/picard_CollectRnaSeqMetrics.xml [hunk 1]
```diff
@@ -22,18 +22,16 @@
       ## The awk line below converts a file obtained from UCSC as specified in the tool help to refFlat format
 
       #if str($gene_reference_source.gene_reference_source_selector) == "gtf"
-        #if $gene_reference_source.refFlat.ext != 'gff3'
-            gtfToGenePred -genePredExt '${gene_reference_source.refFlat}' refFlat.tab.raw &&
-        #else
+        #if $gene_reference_source.refFlat.ext == 'gff3'
             gff3ToGenePred '${gene_reference_source.refFlat}' refFlat.tab.raw &&
+	#else
```
Comment (bgruening) [inline tools/picard/picard_CollectRnaSeqMetrics.xml]: indentation ...

**Comment by bgruening** [comment]
ping @abretaud

**Pull request merged**